lib/Doctrine/ORM/Mapping/Driver/DatabaseDriver.php

```diff
@@ -384,21 +386,21 @@ private function buildFieldMappings(ClassMetadataInfo $metadata)
      *
      * @param string $tableName
      *
-     * @psalm-return array{
-     *                   fieldName: string,
-     *                   columnName: string,
-     *                   type: string,
-     *                   nullable: bool,
-     *                   options?: array{
-     *                       unsigned?: bool,
-     *                       fixed?: bool,
-     *                       comment?: string,
-     *                       default?: string
-     *                   },
-     *                   precision?: int,
-     *                   scale?: int,
-     *                   length?: int|null
-     *               }
+     * @return array{
```

Comment:
PHPStorm won't understand that. @psalm-return seemed correct

Reply:
I know it currently is not understood by most IDEs. The problem is to find a return type which satisfies phpcs. `array` is not enough. I'll try `mixed[]`.

Reply:
yeah, phpcs just forbids array but mixed[] is always good